reactrouter/build.sbt

```diff
@@ -0,0 +1,7 @@
+enablePlugins(ScalaJSPlugin)
+
+name := "slinky-react-router"
+
+libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value
```

Comment:
What is this dependency needed for? It should be introduced transitively by the dependency on `core`.

Reply:
:heavy_check_mark: